src/EFCore.PG/Internal/NpgsqlLoggerExtensions.cs

```diff
@@ -12,14 +13,11 @@ public static class NpgsqlLoggerExtensions
         {
             var definition = NpgsqlResources.LogMissingSchema(diagnostics);
 
-            var warningBehavior = definition.GetLogBehavior(diagnostics);
-            if (warningBehavior != WarningBehavior.Ignore)
+            if (diagnostics.ShouldLog(definition))
             {
-                definition.Log(
-                    diagnostics,
-                    warningBehavior,
-                    schemaName);
+                definition.Log(diagnostics, schemaName);
             }
```

Comment:
It's one liner in the end. Why curlies around the call?

Reply:
Just because it's EF Core style :) Am removing.